tango/device_server.py

```diff
@@ -2795,4 +2807,4 @@ def device_server_init(doc=True):
         __doc_MultiAttribute()
         __doc_MultiClassAttribute()
         __doc_UserDefaultAttrProp()
-        __doc_Attr()
+        __doc_Attr()
```

Comment:
Can we get a new line at the end of the file, please?

Reply:
Iv got a weird result here.
I still have a change when i remove or when i add the new line.

Reply:
OK, then you can leave it as is.